spec.html

```diff
@@ -30780,6 +30780,7 @@
         <emu-alg>
           1. Let _obj_ be ? ToObject(*this* value).
           1. Let _len_ be ? ToLength(? Get(_obj_, `"length"`)).
+          1. If _comparefn_ is not *undefined* and IsCallable(_comparefn_) is *false*, throw a *TypeError* exception.
```

Comment:
This should be step 1 instead of step 3, since the other steps aren’t needed in case we throw.